packages/compat/src/compat-adapters/ember-source.ts

```diff
@@ -115,6 +115,13 @@ export default class extends V1Addon {
       trees.push(new FixCycleImports([packages]));
     }
 
+    // As long as the inspector support is not implemented in an actual
+    // ember-source release, we can't set a range. This fix should be
+    // included for any version that has Vite support starting 3.28.
+    if (satisfies(this.packageJSON.version, '>=3.28')) {
```

Comment:
turns out that semver isn't considering beta or canary as greater than 3.28 🫠 I figure we should remove this check but leave a comment that when we add support upstream we should wrap the tree in a condition 👍

Reply:
Ah Ok 🫠 I pushed this at the end of my work day, I didn't have a look yet. Let's remove that condition then ^^